src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,69 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as currentTodoActions } from '../../features/currentTodo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
 export const TodoModal: React.FC = () => {
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+  const [user, setUser] = useState<User | null>();
```

Comment:
set initiall value for this state
```suggestion
  const [user, setUser] = useState<User | null>(null);
```